docs/CLI.md

```diff
@@ -52,64 +54,59 @@ cov versions
 cov serve
 ```
 * Starts up the ng server with needed flags
-* Same as `ng serve --proxy-config proxy.conf.json`
+* Same as `ng serve --proxy-config proxy.conf.js --sourceMap=true`
 
 ```
-cov serve-e2e
+cov serve-prod
 ```
 * Starts the ng server with flags needed to run e2e coverage
-* Same as `node --max_old_space_size=5048 ./node_modules/.bin/ng serve --aot --prod --sourcemap=true --build-optimizer --proxy-config proxy.conf.json`
+* Same as `ng serve --proxy-config proxy.conf.js --sourceMap=true --aot --prod --optimization`
 
 ```
 cov e2e
 ```
 * Runs the end to end tests against local code base
-* Same as `ng e2e --config --proxy-config e2e/proxy.conf.json`
+* Same as `ng e2e`
 
 ```
 cov e2e-for-coverage
 ```
 * Runs the end to end tests against the e2e proxy
-* Same as `ng e2e -pc proxy.conf.json --no-serve --base-href=http://localhost:5050`
+* Same as `ng e2e --baseUrl=http://localhost:5050 --devServerTarget=`
```

Comment:
don't we still need the proxy conf on here?

Reply:
Same as above.